blockchain/chaingen/generator.go

```diff
@@ -1078,6 +1078,13 @@ func (g *Generator) CalcNextRequiredStakeDifficulty() int64 {
 	return nextDiff
 }
 
+// CalcNextRequiredDifficulty returns the required stake difficulty (aka
+// ticket price) for the block after the current tip block the generator is
+// associated with.
```

Comment:
Since this is the exported (and hence externally visible) function, please move the existing comment to this function, and instead modify the unexported function comment to be this plus:

```Go
// ...
//
// See the documentation of CalcNextRequiredStakeDifficulty for more details.